ckanext/datatablesview/templates/datatables/datatables_view.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block page %}
-  {% set filters = request.params.get('filters', '')|e if request.params is defined else request.args.get('filters', '')|e %}
+  {% set filters = url_quote(request.params.get('filters', '').encode('utf-8')).encode('ascii')|e if request.params is defined else url_quote(request.args.get('filters', '').encode('utf-8')).encode('ascii')|e %}
```

Comment:
instead of adding a new global function to all templates used everwhere, turn this dense bit of mostly-python code into a template helper function in the datatables extension.